docs/guide/dev/patching/README.md

```diff
@@ -206,11 +206,11 @@ If the build that you used had spell checker plugins (`scayt` or `wsc`), then yo
 
     <img src="%BASE_PATH%/assets/img/patching_05.png" width="391" height="181" alt="Plugin revision in GitHub">
```

Comment:
I'm not sure if it wouldn't be better to update the screen to remove the WSC plugin submodule from it :thinking: 

Also, this method does no longer works as we now load SCAYT as npm dependency.

Reply:
Yeah, you are right. I didn't notice how outdated this guide is. Anyway, I would be for extracting it as a separate issue so we will be able to test these changes during the testing phase, WDYT?

Regarding the screenshots themselves, fortunately, there is also `scayt` on them so I think we can just move that part also to the separate issue addressing the whole guide.

Reply:
Extracted https://github.com/ckeditor/ckeditor4/issues/5117